packages/ui/src/common/utils/validation.tsx

```diff
@@ -63,7 +63,7 @@ export const minContext = (
     }
 
     const validationValue = new BN(get(this.options.context, contextPath))
-    if (validationValue && validationValue.gt(new BN(value))) {
+    if (validationValue && validationValue.gte(new BN(value))) {
```

Comment:
Same thing here

Reply:
Fixed in [46462da](https://github.com/Joystream/pioneer/pull/3793/commits/46462da0b5188b7e92efb660b0ff707266d56083)